landlab/components/flow_accum/flow_accumulator.py

```diff
@@ -987,31 +984,40 @@ def accumulate_flow(self, update_flow_director=True):
             self._grid["node"]["flow__data_structure_delta"][:] = delta[1:]
 
             if self._is_raster:
-                tempD = BAD_INDEX_VALUE * np.ones((self._grid.number_of_links * 2))
+                tempD = BAD_INDEX_VALUE * np.ones(
```

Comment:
@SiccarPoint FYI. This is the janky approach I had to take because we can't store information at D8s...

Reply:
Thanks...